### ISSUE TITLE
Build Failed orcid.connector.url

### ISSUE THREAD

**Issue by shoutinblues** [atmire/ORCID-rest-api#1]
I followed the installation instructions for our DSpace 6.3 system, but "ant update" fails with this message:

BUILD FAILED
/ebs/dspace-source/dspace-6.3-release/dspace/target/dspace-installer/build.xml:75: Property orcid.connector.url was circularly defined.

When I ran the check, it complained about a few things that didn't seem to be critical.  Including them below in case they are relevant.

[glenn@rctest dspace-6.3-release]$ git apply --check ./dspace_6x…stable_6x.diff
error: cannot apply binary patch to 'dspace/modules/xmlui-mirage2/src/main/webapp/themes/Mirage2/images/orcid_icon.png' without full index line
error: dspace/modules/xmlui-mirage2/src/main/webapp/themes/Mirage2/images/orcid_icon.png: patch does not apply
error: dspace/modules/xmlui-mirage2/src/main/webapp/themes/Mirage2/xsl/theme.xsl: already exists in working directory
error: cannot apply binary patch to 'dspace/modules/xmlui/src/main/webapp/themes/Mirage/images/orcid_icon.png' without full index line
error: dspace/modules/xmlui/src/main/webapp/themes/Mirage/images/orcid_icon.png: patch does not apply

Would appreciate any ideas on how to get this patch applied.

**Comment by jonas-atmire** [comment]
@shoutinblues 
This ‘circularly defined’ error is indeed because of a ‘missing’ property.

Later on the actual patch could be updated so that this configuration is always set to the property mentioned later on.

In the meantime, this can be fixed in 2 ways:
1. Either you can set the correct properties in the build profile: the following property can be used `<orcid.connector.url>https://orcid.org/</orcid.connector.url>`
2. This config can also be put in the dspace.cf file itself `orcid.connector.url= https://orcid.org/`

**Comment by shoutinblues** [comment]
Thank you for the advice.  I was unable to determine which file to edit for solution #1, so proceeded to #2.  I modified the dspace.cfg file in the installer directory, but the error did not go away.  I removed the change from dspace.cfg and added it to local.cfg instead, and this worked.   For others who may wonder, the local.cfg file I edited appeared in [dspace-source]/dspace/target/dspace-installer/local.cfg after running "mvn -U clean package".
